### ISSUE TITLE
no ability to access dropdowns

### ISSUE THREAD

**Issue by delanym** [syscolabs/kasaya#19]
I can't access dropdown menus. I tried clicking the "down arrow" and pressing "down arrow" or "page down".

**Comment by AshanthaLahiru** [comment]
At the moment Kasaya is searching the elements based on their texts. You can achieve this by providing an aria-label to the particular element .

Example:
```
click "sample-aria-label"
```

**Comment by mthompson36** [comment]
Will there be another way to select a dropdown option when the element doesn't have a `aria-label`?

**Comment by AshanthaLahiru** [comment]
@mthompson36 , At the moment Kasaya is considering following attributes of an element when it comes to clicking,
 - NAME
 - TITLE
 - TYPE
 - VALUE
 - PLACEHOLDER
 - ARIA LABEL

If your element has any above attribute, you can click using that attribute value.
